front/app/components/SeatInfo/BillingInfo.tsx

```diff
@@ -83,7 +83,7 @@ const BillingInfo = ({ seatType }: SeatInfoProps) => {
   const remainingSeats = totalSeats - usedSeats;
 
   let totalSeatsBreakdownMessage = formatMessage(messages.seatsWithinPlanText);
-  if (usedSeats > maximumSeatNumber) {
+  if (additionalSeats) {
```

Comment:
Fix to show breakdown of seats when there are additional seats

Reply:
Shouldn't our tests have caught this?

Reply:
There is one but it is handling the case where seats are extra. Happy to add one for a scenario where the seats are not extra